### ISSUE TITLE
PS2 PAL - After reconnecting the controller, entering mc9 will instantly exit back out of it

### ISSUE THREAD

**Issue by olokos** [israpps/wLaunchELF_ISR#13]
Hey,

Great work so far, thanks!

There is a bug on PS2 PAL 75004 with original DS2 controller (DS1 is the same I think, but I'm not sure 100%).

When wLe is started and shows list of devices, if the controller is disconnected and reconnected again - entering mc0 will automatically exit out of mc0.

From what I've gathered it might be related to commits 33383da9216407140bfd91512526b0c9e9a79471 and 4d0d96aa8dced0061bc8eaa88386f83adf8f5f5c .

Not sure where the issue is exactly, but it's pretty annoying when used with wireless controller, as those disconnect after a while of idling automatically.

Do you remember where the problem could possibly be @parrado ?

**Comment by parrado** [comment]
Thanks for reporting this issue. Have you replicated it on a different console (NTSC unit)? this could be timing related. Anyway, I'll track it down once I have enough time.

**Comment by olokos** [comment]
I only have 2 PAL consoles, can't really test it on NTSC, unless MechaPwn counts.

**Comment by israpps** [comment]
> Thanks for reporting this issue. Have you replicated it on a different console (NTSC unit)? this could be timing related. Anyway, I'll track it down once I have enough time.

Yesterday I suffered this issue, ir automatilly got out of mc0

I was incapable of replicating this issue tho

**Comment by Pacorretaco** [comment]
Yeah I experienced it too after the PADEMU implementation on 27th of February and reported it but not here. Probably remember 

I think it may be somehow related with the connected devices, like USB devices. For some reason it will happen or not happen depending on what is connected

**Comment by israpps** [comment]
Temporary workaround:
https://github.com/israpps/wLaunchELF_ISR/tree/preds34

**Comment by olokos** [comment]
This bug also happens with 3rd party wireless controllers for PS2 and is in fact fixed/not broken yet in PreDS3/4 build

**Comment by israpps** [comment]
confirmed this issue also happens when opening `MISC/Debug Info`

**Comment by israpps** [comment]
corrected title because this also happened to me on 70011 and 18000

**Comment by Windows10User2** [comment]
This issue hasn't been fixed yet.

**Comment by Windows10User2** [comment]
> @Windows10User2, this issue is only in DS34 versions. Since you have Brook Super Converter, use any version without DS34 in the name.

Does it only happen with PAL models and if having USB devices connected?
